corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -81,10 +82,23 @@ public void listTables(String namespace)
     {
         log.info("\n=====Tables=======\n");
         for (CorfuStoreMetadata.TableName tableName : runtime.getTableRegistry()
-            .listTables(namespace)) {
+                .listTables(namespace)) {
             log.info("Table: " + tableName.getTableName());
             log.info("Namespace: " + tableName.getNamespace());
         }
         log.info("\n======================\n");
     }
+
+    /**
+     * Print information about a specific table in CorfuStore
+     */
+    public void printTableInfo(String namespace, String tablename) {
+        log.info("\n======================\n");
```

Comment:
what if namespace or/and table doesn't exist?

Reply:
Good point, I think its better to detect and fail gracefully. fixed in latest patch. thanks